themes/rad/layouts/index.html

```diff
@@ -4,25 +4,25 @@
     <div id="peer">
       <h2>Peer–to–peer</h2>
       <p>
-        The Radicle stack builds on IPFS, which means it works without a centralized server. No hassle with setting up
-        your own hosting, and no need to trust companies with your data.
+        Radicle is a decentralized, collaborative revision control system built on IPFS.
```

Comment:
@geigerzaehler's point is a good one. It is quite misleading to name it a collaborative revision control system. (Also, the only use of that term that I know of is RCS - I think people now use VCS?)